### PR TITLE
[#1882757] Conflicting component name 'RunManager'

### PR DESCRIPTION
**Pull request by wangmingliang-ms**
[AB#1882757](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1882757)
#5888

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-lib/src/main/java/com/microsoft/azure/toolkit/intellij/connector/AzureServiceResource.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public synchronized T getData() {
 
     @Override
     public Map<String, String> initEnv(Project project) {
-        return this.definition.initEnv(this.data, project);
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-lib/src/main/java/com/microsoft/azure/toolkit/intellij/connector/AzureServiceResource.java:54]: add `@Nullable` and `@Nonnull` to `data` and `getData()`?

**Review by Flanker32 — APPROVED** [review]
LGTM

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-lib/src/main/java/com/microsoft/azure/toolkit/intellij/connector/AzureServiceResource.java (hunk 1 above)
Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-lib/src/main/java/com/microsoft/azure/toolkit/intellij/connector/AzureServiceResource.java:54]: not at this stage

**Review by andxu — APPROVED** [review]
LGTM

**Pull request merged**